### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by cucumber007**
Apply my changes plz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# test
\ No newline at end of file
+# test
+
+Changes
```
Comment (original-spqrta) [inline README.md]: Do some fixes

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# test
\ No newline at end of file
+# test
+
```
Comment (original-spqrta) [inline README.md]:
```suggestion
Addition
```

File: README.md (hunk 1 above)
Comment (cucumber007) [reply README.md]: @original-spqrta ok

**Comment by cucumber007** [comment]
@original-spqrta how to do that fixes?

**Comment by cucumber007** [comment]
@original-spqrta hey

**Comment by original-spqrta** [comment]
@original-spqrta suka blyat

**Comment by cucumber007** [comment]
@original-spqrta suka blyat

**Pull request merged**